doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md

```diff
@@ -125,10 +125,19 @@ Install facedetect for [image thumbnails focal point](../../04_Assets/03_Working
 apt-get install facedetect
 ```
 
-# html2text
+## html2text
 
 Install html2text for [Pimcore mail](../../19_Development_Tools_and_Details/25_Email_Framework/01_Pimcore_Mail.md)
 
 ```bash
 apt-get install html2text
 ```
+
+# Check your installation
+
+You can check if everything is here in the UI, via the `Tools` / `System Info & Tools` / `System-Requirements Check` page.  
```

Comment:
You can check system requirements, via Admin UI `Tools` / `System Info & Tools` / `System-Requirements Check` menu. 

Or via following CLI command: